.agent-os/cache/optimization-helpers.js

```
@@ -0,0 +1,241 @@
+const CONTEXTS_PATH = '/Users/nathanvale/code/bun-changesets-template/.agent-os/cache/common-contexts.txt';
+
+/**
+ * GET_COMMON_CONTEXTS - Returns array of file paths that should be pre-loaded
+ * @returns {string[]} Array of absolute file paths to pre-load
+ */
+function GET_COMMON_CONTEXTS() {
+    try {
+        const manifest = JSON.parse(fs.readFileSync(MANIFEST_PATH, 'utf8'));
+        return manifest.optimization_plan.contexts_to_preload;
+    } catch (error) {
+        console.error('Failed to load common contexts:', error.message);
```

ESLint

[error] 'console' is not defined.

(no-undef)